cdap-docs/admin-manual/source/hadoop-compatibility.rst

```diff
@@ -118,7 +118,7 @@ Hadoop components, as required:
 +                +---------------------------------+---------------------------------------+
 |                | |cdh-title|                     | |cdh-versions| |note-4|               |
 +                +---------------------------------+---------------------------------------+
-|                | Hortonworks Data Platform (HDP) | |hdp-versions| |note-4|               |
+|                | Hortonworks Data Platform (HDP) | 2.0 through 2.6.4 |note-4|            |
```

Comment:
Un intended change, please remove.

Reply:
This change is intentional because HDP  has different spark versions for each patch release. 2.6.5 uses spark 2.3 which we dont support.